Makefile

```diff
@@ -1,5 +1,44 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
+RAS_REPOS="ras-party" "ras-secure-message" "ras-frontstage" "ras-collection-instrument" "django-oauth2-test" "ras-rm-auth-service" "respondent-home-ui" "response-operations-ui" "response-operations-social-ui" "sdc-uaa"
+RM_REPOS="rm-sample-service" "rm-case-service" "rm-action-service" "rm-actionexporter-service" "iac-service" "rm-sdx-gateway" "rm-collection-exercise-service" "rm-survey-service" "rm-notify-gateway" "rm-comms-template-service" "rm-reporting"
+REPOS=${RAS_REPOS} ${RM_REPOS}
+REPOS=${RAS_REPOS} ${RM_REPOS}
+
+check-env:
+ifndef RASRM_HOME
+	$(error RASRM_HOME environment variable is not set.)
+endif
+	@ printf "\n[RASRM_HOME set to ${RASRM_HOME}]\n"
+
+clone: check-env
+	@ printf "\n[Cloning into ${RASRM_HOME}]\n"
+	@ for r in ${REPOS}; do \
+		echo "($${r})"; \
+		if [ ! -e ${RASRM_HOME}/$${r} ]; then \
+			git clone git@github.com:ONSdigital/$${r}.git ${RASRM_HOME}/$${r}; \
+		else \
+			echo "  - already exists: skipping"; \
+		fi; echo ""; \
+	done
+
+build: clone
+	@ printf "\n[Building ${RASRM_HOME}]\n"
+	@ for r in ${REPOS}; do \
+		echo "Building ($${r})"; \
```

Comment:
Would it be worth adding a branch name as well similar to what SDX does?

Reply:
where does SDX do that?

Reply:
Ignore me I was wrong, sdx-compose shows the branch name on the `make update` not the build https://github.com/ONSdigital/sdx-compose/blob/master/Makefile#L40

Reply:
I have added this to the build command now